src/ontology/uberon.Makefile

```diff
@@ -1004,784 +816,470 @@ subsets/life-stages-minimal.owl: subsets/merged-partonomy.owl
-#	owltools --catalog-xml catalog-nlx.xml $< --abox-to-tbox  --reasoner-query -r elk -d  -l "$*" -c $(URIBASE)/nlx/neuron  -o $@
-#	owltools --catalog-xml catalog-nlx.xml $<  --reasoner-query -r elk -d -l "$*" --abox-to-tbox  --make-ontology-from-results $(URIBASE)/nlx/neuron.owl -o $@
 
-# ----------------------------------------
-# Rules
-# ----------------------------------------
-# We got rid of this as per discussion with @cmungall
-#ipo.obo: uberon.obo
-	#blip-findall  -i $< -consult util/partof.pro new_part_of/2 -label -no_pred -use_tabs | sort -u | $(SCRIPTSDIR)/tbl2obolinks.pl  --rel part_of --source reference_0000032 - > $@
+# Basic subset
+# FIXME: https://github.com/obophenotype/uberon/issues/3013
+basic.owl:  uberon-basic.owl
+	$(ROBOT) merge -i $< annotate --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) -o $@
 
-# ----------------------------------------
-# REPORTING
-# ----------------------------------------
+basic.obo: basic.owl
+	$(MAKEOBO)
 
-# DOCS
-# We got rid of this as per discussion with @cmungall
-#relation_table.txt:
-#	echo "STRONG WARNING: $@ skipped, because there is no more blip." && touch $@
-#	#blip-findall -r uberon -consult util/relation_report.pro "row(R)" -select R > relation_table.txt
+# Cumbo subset
+subsets/cumbo.owl: basic.owl
+	$(OWLTOOLS) $< --extract-ontology-subset --subset cumbo --iri $(URIBASE)/uberon/$@ -o $@
 
-# number of uses of each relation.
-# We got rid of this as per discussion with @cmungall
-#%-relstats: %.obo
-#	echo "STRONG WARNING: $@ skipped, because there is no more blip." && touch $@
-#	#blip-findall -r uberon  "aggregate(count,X-T,parent(X,R,T),Num)" -select "R-Num" -no_pred | | sort -nk2 > $@
+subsets/cumbo.obo: subsets/cumbo.owl
+	$(MAKEOBO)
 
-#%-el.owl: %.owl
-#	makeElWithoutReasoning.sh -i `pwd`/$< -o `pwd`/$@
+# Common anatony subset
+common-anatomy.owl: $(ONT).owl
+	$(OWLTOOLS) $< --extract-ontology-subset --fill-gaps --subset common_anatomy \
+		    -o $@.tmp.owl && mv $@.tmp.owl $@ && \
+	$(ROBOT) annotate --input $@ --ontology-iri $(ONTBASE)/$@ $(ANNOTATE_ONTOLOGY_VERSION) \
+		          -o $@.tmp.owl && mv $@.tmp.owl $@
+.PRECIOUS: common-anatomy.owl
 
-# ----------------------------------------
-# SPELLCHECK
-# ----------------------------------------
+subsets/immaterial.obo: uberon.owl
+	$(OWLTOOLS) $< --reasoner-query -r elk -d  UBERON_0000466 \
+		    --make-ontology-from-results $(URIBASE)/uberon/$@ \
+		    -o -f obo $@ --reasoner-dispose 2>&1 > $@.LOG
 
-# notes:
-# to add to a dict
-# yes a | aspell check --home-dir . obo-syntax.html
-#
-# setting --home-dir to . will cause the dictionary .aspell.en.pws  in this directory to be used
+subsets/metazoan-view.owl: basic.owl
+	cp $< $@
 
-# interactive
-ispellcheck: $(SRC)-ispellcheck
+# The first step is a simple "merge+reason", but it still requires
+# Owltools because ROBOT has no equivalent to the -x option to simply
+# ignore unsatisfiable classes without erroring out.
```

Comment:
Does not seem like a good reason to keep it, at least not reading it this way.. Since ROBOT reason is used in the next step, what is the reason to maintain this?

Reply:
My bad, the comment is not clear enough. `-x` does not simply _ignore_ the unsatisfiable classes, it actively _removes them_ – so `robot reason` does not ever see the problematic classes. If we removed the Owltools step here, and the subset happens to contains unsats, the entire step would fail.